docs/src/intro/use_cases.md

```diff
@@ -2,16 +2,16 @@
 
 ## [Wflow models for the Meuse and Rhine](@id case_rws)
 
-Reliable hydrological models for the Rhine and the Meuse river basin are necessary for
+Reliable hydrological models for the Rhine and the Meuse river basins are necessary for
 short-term forecasting of river flows and long-term predictions for strategic water
-management planning. In collaboration with Rijkswaterstaat, Deltares is setting up a new
+management planning. In collaboration with Rijkswaterstaat, Deltares is developing a new
 line of models for the Rhine and the Meuse basins. The models will be used for forecasting
 and to estimate the impact of climate change on water resources and extreme streamflow. In
-the model development, we seek to improve hydrological predictions by including relevant
-processes in the model schematization. The modularity of the wflow framework is ideal for
+the model development, we aim to improve hydrological predictions by including relevant
+processes in the model schematization. The modularity of the Wflow framework is ideal for
```

Comment:
We decided to use lowercase for `wflow` unless at the start of a sentence or when we mention the Wflow.jl framework (also the case for the GMD paper about the wflow_sbm model).

```suggestion
processes in the model schematization. The modularity of the wflow framework is ideal for
```